pkg/yurthub/cachemanager/storage_wrapper.go

```diff
@@ -145,7 +156,18 @@ func (sw *storageWrapper) List(key string) ([]runtime.Object, error) {
 	}
 
 	for i := range bb {
-		obj, gvk, err := sw.backendSerializer.Decode(bb[i], nil, nil)
+		//get the gvk from json data
+		gvk, err := json.DefaultMetaFactory.Interpret(bb[i])
```

Comment:
How about move  `Interpret` out of `for` loop, so run `Interpret` only one time.

Reply:
code is already updated, thanks for review